MovieQuiz/Models/GameRecord.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  GameRecord.swift
+//  MovieQuiz
+//
+//  Created by Toto Tsipun on 14.11.2022.
+//
+
+import Foundation
+
+struct GameRecord: Codable, Comparable {
+    
+    var correct: Int
+    let total: Int
+    let date: Date
+    
+    static func < (lhs: GameRecord, rhs: GameRecord) -> Bool {
+        if lhs.correct < rhs.correct {
+            return true
+        } else {
+            return false
+        }
```

Comment:
Можно возвращать сразу результат вычисления выражения:
```suggestion
        return lhs.correct < rhs.correct
```

А еще, если функция однострочная, можно опускать return:
```suggestion
        lhs.correct < rhs.correct
```

Reply:
Круто, знала это, но почему-то даже не вспомнила, пока что все усложняю и топорно делаю =(( 
А как сразу красивее и лаконичнее код смотрится))

Reply:
Лучше иногда топорно, зато верно :) Не переживай, это с опытом и насмотренностью придет